cmd/minikube/cmd/get_openshift_versions.go

```diff
@@ -26,8 +26,9 @@ import (
 // getVersionsCmd represents the ip command
 var getVersionsCmd = &cobra.Command{
 	Use:   "get-openshift-versions",
-	Short: "Gets the list of available OpenShift versions available for minishift.",
-	Long:  `Gets the list of available OpenShift versions available for minishift.`,
+	Short: "Gets the list of OpenShift versions available for Minishift.",
+	Long:  `Gets the list of OpenShift versions available for Minishift.`,
+	//MOREINFO: Do we want to elaborate on usage or examples? Like is this STDOUT only, can we output to a file, do we want to pass this onto something else?
```

Comment:
How do you want to handle these _MOREINFO_s? Some of these things have some related issues. Is the plan to merge and keep some of these comments? Or are you planning to address them all prior to merging? I think for some of your comments, it makes more sense to create trackable issues, instead of having some comments in the code. Regarding this particular issue, it maps to https://github.com/minishift/minishift/issues/197